docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -6,6 +6,140 @@ description: "Review which adjustments must be made to migrate from Camunda 8.7.
 
+## Camunda Exporter
+
+The new Camunda Exporter brings the importing and archiving logic of web components (Tasklist and Operate) closer to the distributed platform (Zeebe). This simplifies the installation, enables scalability for the web applications, reduces latency when showing runtime and historical data, and reduces data duplication (resource consumption).
+
+![87-state](./img/87-state-refined.png)
+
+Camunda web applications use importers and archivers to consume, aggregate, and archive historical data provided by the Elasticsearch (ES) or OpenSearch (OS) exporters. The new Camunda Exporter introduces changes in the architecture users should be aware of.
+
+During the migration/update phase, the Operate and Tasklist importer and the Elasticsearch exporter are still required. This allows 8.7 data written to the Zeebe journal, or already in the Zeebe Elasticsearch indices, to be imported into the new harmonized index schema.
+
+![brownfield](./img/target-brown-field-refined.png)
+
+### Harmonized Indices
+
+The harmonized schema looks like the following:
+
+![harmonized-schema](./img/harmonized-indices-miro-v2.png)
+
+This reduces the need for several indices and to make use of existing ones (Operate indices are used by Tasklist, etc.)
+
+### Notes for updating to 8.8
+
+When updating from Camunda 8.7 to 8.8, the following should be considered regarding the Camunda Exporter and harmonized indices.
+
+#### Ensure the exporter/import backlog is small
+
+For data from 8.7 to be imported, the importers must run first and complete before the Camunda Exporter can start.
+
+The importer reads data from existing Zeebe Elasticsearch indices, and when it detects data from newer versions, it marks itself as complete. The Camunda Exporter then starts exporting only when the importer is done.
+
+:::important
+To reduce the potential of building a backlog,
+the user needs to make sure to drain the backlog queue as much as possible before the update.
+:::
+
+The backlog can be validated via the following Prometheus metrics:
+
+- `operate_import_time_seconds_sum`
+- `operate_import_time_seconds_count`
+
+For example, the following query shows the import latency. If the result is in a range of seconds, it is fine to update.
+
+```promql
+sum(rate(operate_import_time_seconds_sum{namespace=~"$namespace", partition=~"$partition", pod=~"$pod"}[$__rate_interval]))
+/ sum(rate(operate_import_time_seconds_count{namespace=~"$namespace", partition=~"$partition", pod=~"$pod"}[$__rate_interval]))
+```
+
+#### Data migration
+
+To enable Tasklist to utilize the new harmonized indices, Camunda 8.8 provides a migration application to enhance existing process models. This migration application runs asynchronously to the other applications until the importers are done, and ensures that all processes created in 8.7 are migrated and enhanced.
+
+The migration application can be found in the `camunda/bin/migrate` directory.
```

Comment:
Is there any other step/information required here? Is there a specific point in the update process where this should be run (judging by the positioning, is it after the importers are done, but before you run the prefix migration?). It might be nice to call that out if so:
```suggestion
The migration application can be found in the `camunda/bin/migrate` directory. Ensure the application is run prior to updating any [custom prefixes](#update-any-custom-prefixes).
```

If there is no recommendation, we should state that.

Reply:
The suggestion is not correct. The index prefix migration is only necessary if they customer used a custom prefix before and then IT MUST run as a first step. I just have this moved more to the back, as it is not always needed.